ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Tests/Uniform.hs

```diff
@@ -55,11 +56,11 @@ tests =
 
 makeProperty ::
   GenesisTest ->
-  Int ->
+  Peers PeerSchedule ->
   StateView ->
   [PeerId] ->
   Property
```

Comment:
Having the property rely on the `Peers PeerSchedule` instead of a `PointSchedule` will be incompatible with https://github.com/IntersectMBO/ouroboros-consensus/pull/842. I'm sure we can find a way to make https://github.com/IntersectMBO/ouroboros-consensus/pull/842 allow that though.